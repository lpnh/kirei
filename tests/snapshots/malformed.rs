#[test]
fn unclosed_paragraph() {
    format_and_snapshot!("malformed/unclosed_paragraph.html");
}

#[test]
fn unclosed_div() {
    format_and_snapshot!("malformed/unclosed_div.html");
}

#[test]
fn unclosed_block() {
    format_and_snapshot!("malformed/unclosed_block.html");
}
