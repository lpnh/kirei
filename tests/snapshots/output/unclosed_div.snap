---
source: tests/snapshots/malformed.rs
description: "<main> <div> <h2>Test</h2> <p> Where did that div go </p> </main>\n"
---
<main>
    <div>
        <h2>Test</h2>
        <p>Where did that div go</p>
</main>
